e2e-tests/gef/cypress/fixtures/MOCKS/MOCK_DEALS.js

```diff
@@ -0,0 +1,214 @@
+    'user-status': 'active',
+    _id: '619bae3467cc7c002069fc21',
+  },
+};
+
+const MOCK_FACILITY_ONE = {
+  type: 'CASH',
+  hasBeenIssued: false,
+  name: 'Facility one',
+  shouldCoverStartOnSubmission: true,
+  coverStartDate: 1638403200000,
+  coverEndDate: '2030-01-01T00:00:00.000Z',
+  monthsOfCover: 20,
+  details: [],
+  detailsOther: '',
+  currency: 'GBP',
+  value: 2000,
+  coverPercentage: 80,
+  interestPercentage: 1,
+  paymentType: 'IN_ADVANCE_MONTHLY',
+  createdAt: 1638363596947,
+  updatedAt: 1638442632540,
+  ukefExposure: 1600,
+  guaranteeFee: 0.9,
+  submittedAsIssuedDate: '1638363717231',
+  feeType: 'in advance',
+  feeFrequency: 'Monthly',
+  dayCountBasis: 365,
+  coverDateConfirmed: false,
+  changedToIssued: null,
+};
+
+const MOCK_FACILITY_TWO = {
+  type: 'CASH',
+  hasBeenIssued: true,
+  name: 'Facility two',
+  shouldCoverStartOnSubmission: true,
+  coverStartDate: 1638403200000,
+  coverEndDate: '2030-01-01T00:00:00.000Z',
+  monthsOfCover: null,
+  details: [],
+  detailsOther: '',
+  currency: 'GBP',
+  value: 2000,
+  coverPercentage: 80,
+  interestPercentage: 1,
+  paymentType: 'IN_ADVANCE_MONTHLY',
+  createdAt: 1638363596947,
+  updatedAt: 1638442632540,
+  ukefExposure: 1600,
+  guaranteeFee: 0.9,
+  submittedAsIssuedDate: '1638363717231',
+  ukefFacilityId: '0030113305',
+  feeType: 'in advance',
+  feeFrequency: 'Monthly',
+  dayCountBasis: 365,
+  coverDateConfirmed: true,
+  changedToIssued: null,
+};
+
+const MOCK_FACILITY_THREE = {
+  type: 'CASH',
+  hasBeenIssued: false,
+  name: 'Facility three',
+  shouldCoverStartOnSubmission: true,
+  coverStartDate: 1638403200000,
+  coverEndDate: '2030-01-01T00:00:00.000Z',
+  monthsOfCover: 30,
+  details: [],
+  detailsOther: '',
+  currency: 'GBP',
+  value: 2000,
+  coverPercentage: 80,
+  interestPercentage: 1,
+  paymentType: 'IN_ADVANCE_MONTHLY',
+  createdAt: 1638363596947,
+  updatedAt: 1638442632540,
+  ukefExposure: 1600,
+  guaranteeFee: 0.9,
+  submittedAsIssuedDate: '1638363717231',
+  ukefFacilityId: '0030113305',
+  feeType: 'in advance',
+  feeFrequency: 'Monthly',
+  dayCountBasis: 365,
+  coverDateConfirmed: false,
+  changedToIssued: null,
+};
+
+const MOCK_FACILITY_FOUR = {
+  type: 'CASH',
+  hasBeenIssued: false,
+  name: 'Facility four',
+  shouldCoverStartOnSubmission: true,
+  coverStartDate: 1638403200000,
+  coverEndDate: '2030-01-01T00:00:00.000Z',
+  monthsOfCover: 6,
+  details: [],
+  detailsOther: '',
+  currency: 'GBP',
+  value: 2000,
+  coverPercentage: 80,
+  interestPercentage: 1,
+  paymentType: 'IN_ADVANCE_MONTHLY',
+  createdAt: 1638363596947,
+  updatedAt: 1638442632540,
+  ukefExposure: 1600,
+  guaranteeFee: 0.9,
+  submittedAsIssuedDate: '1638363717231',
+  ukefFacilityId: '0030113305',
+  feeType: 'in advance',
+  feeFrequency: 'Monthly',
+  dayCountBasis: 365,
+  coverDateConfirmed: false,
+  changedToIssued: null,
+
+};
+
+const MOCK_USER_MAKER = {
+  username: 'BANK1_MAKER1',
```

Comment:
you don't need the `userId` property here?

Reply:
No, it breaks the email call doing that.  It seems to make its own userID/ignores it if its blank for e2e.  With a userId, it draws an error that cannot find it